File: tools/dbench/answers/go_haiku/s2/r1_orders.go
package main

import (
	"encoding/json"
	"fmt"
	"os"
)

type Order struct {
	Id     int     `json:"id"`
	Amt    float64 `json:"amt"`
	Status string  `json:"status"`
}

func main() {
	data, _ := os.ReadFile("orders.json")
	var orders []Order
	json.Unmarshal(data, &orders)

	paid := 0.0
	open := 0

	for _, o := range orders {
		if o.Status == "paid" {
			paid += o.Amt
		} else if o.Status == "open" {
			open++
		}
	}

	fmt.Println(paid)
	fmt.Println(open)
}

File: tools/dbench/answers/python_sonnet/s3/r1_orders.py
import json

with open("orders.json") as f:
    orders = json.load(f)

paid = [o for o in orders if o["status"] == "paid"]
open_ = [o for o in orders if o["status"] == "open"]
print(sum(o["amt"] for o in paid))
print(len(open_))
